### DISCUSSION TITLE
Migration Docs: 6.0.0 to 6.1.6

### DISCUSSION THREAD

**Discussion by TheWebbe** [serenity-is/Serenity#6619] [Help]
Not sure if this is the correct place to post this, I was reading through the [Migration->6.0.0 to 6.1.6](https://serenity.is/docs/migration/v6-0-0-to-v6-1-6) doc and all of the file references are for serenity premium and not accessible if you are not subscribed to premium. 

Example: 
Updating appsettings.bundle.json
If you never modified appsettings.bundles.json you can replace the whole file with [this file](https://github.com/serenity-premium/startsharp/blob/6a35bac28dccd4eb7720cf1094e330b2d6570aea/StartSharp/StartSharp.Web/appsettings.bundles.json). Otherwise apply the changes below.

'this file' points to https://github.com/serenity-premium/startsharp/blob/6a35bac28dccd4eb7720cf1094e330b2d6570aea/StartSharp/StartSharp.Web/appsettings.bundles.json which is inaccessible for non-premium members. 

Files Missing: 
appsettings.bundle.json (appsettings.bundle.json)
tsconfig (Updating Namespaces/tsconfig.json)
tsconfig.json (Add New Root tsconfig.json)
package.json (update packages.json)
tsbuild.js (Adding ESBuild Script)
GridPage.cshtml (Add GridPage.cshtml)
GridPageModel.cs (Add GridPageModel.cs)
GridPageExtensions.cs (Add GridPageExtensions.cs)

Is it possible to have these links updated to the non-premium github?

**Comment by volkanceylan** [comment]
You may see that file from Serene repository, there are directions in that document what changes to do as well if you cant access it. As of writing that doc, Serene was not updated yet, so it was not possible to link it

**Comment by volkanceylan** [comment]
Also see this document with more up to date info:

https://serenity.is/blog/2022/10/15/switching-to-es-modules

**Comment by volkanceylan** [comment]
Checked if we can add links from Serene in the doc, but it is not possible. The reason is added to the migration document, here is for reference:

At the time of writing this document, Serene was not yet updated to ES modules as we wanted to test it with a limited user base. So we had to use sample file links from StartSharp repository, which can only be accessed by the premium members.

Serene was updated to ES modules two months later and you may find some of the files at similar locations if you browse Serene repository using this link:

https://github.com/serenity-is/serene/tree/3668a20a2a1c390d975f39e64774c4888357f958

As we directly switched to ES modules in Serene, skipping `Namespaces` folder etc. and due to some changes occuring in those two months like making tsbuild an NPM package, some of the files mentioned in this document have no match in Serene repository.

We recommend following this blog post for ES modules switch in place of this document as it has more up-to-date information and better directions for Serene (and also for StartSharp):

https://serenity.is/blog/2022/10/15/switching-to-es-modules

You should still follow this document for changes that are not related to ES modules switch.
